core/raftlease/fsm.go

```diff
@@ -153,15 +165,42 @@ type entry struct {
 	duration time.Duration
 }
 
+// response stores what happened as a result of applying a command.
```

Comment:
Maybe mention in the comment that response implements FSMResponse. Or better add a line just above 
var _ FSMResponse = (*response)(nil)

Reply:
Will do!